### ISSUE TITLE
"Paste in place" does not work properly for game objects in the container.

### ISSUE THREAD

**Issue by coeyes** [PhaserEditor2D/PhaserEditor2D-v3#255]
## Version
* Phaser Editor 2D Version: v3.36.0
* Operating System: Windows 10
* Web Browser and version: Chrome 107

## Description

When the coordinates and scale of the container are changed (position is not [0,0]),
"Paste in place (ctrl+shift+v)" does not work properly for game objects in the container.

Pasted object should be located at same position of original.
Please refer to these pictures.

![2022-11-29 00 29 40 -- Img_001](https://user-images.githubusercontent.com/260646/204319934-91c94669-9943-4ad2-a9e0-3fe4dbc84134.jpg)

![2022-11-29 00 29 48 -- Img_001](https://user-images.githubusercontent.com/260646/204319960-9ff79b5e-7d32-4b4f-b4b4-705f2d6020eb.jpg)

![2022-11-29 00 30 45 -- Img_001](https://user-images.githubusercontent.com/260646/204319987-a4579c93-747c-48ec-b4f3-6dbc97c9e0ab.jpg)

**Comment by PhaserEditor2D** [comment]
Yes, I can reproduce the bug. I will work on it. I keep you updated.

**Comment by PhaserEditor2D** [comment]
Hi. Unfortunately, I didn't have the time to fix this issue soon. I hope it is not blocking your game.
